### PR TITLE
[incubator/sentry-kubernetes] Add release/environment values

### PR DESCRIPTION
**Pull request by craigmulligan**
This allows you to customize settings here: https://github.com/getsentry/sentry-kubernetes/blob/master/sentry-kubernetes.py#L27-#L28

**What this PR does / why we need it**:
This add environment variable configurations for the sentry-kubernetes docker image. This allows you to customize settings here: https://github.com/getsentry/sentry-kubernetes/blob/master/sentry-kubernetes.py#L27-#L28

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/sentry-kubernetes/templates/deployment.yaml [hunk 1]
```diff
@@ -26,6 +26,10 @@ spec:
               secretKeyRef:
                 name: {{ template "sentry-kubernetes.fullname" . }}
                 key: sentry.dsn
+          - name: ENVIRONMENT
+            value: {{ .Values.sentry.environment }}
+          - name: RELEASE
+            value: {{ .Values.sentry.release }}
```
Comment (gianrubio) [inline incubator/sentry-kubernetes/templates/deployment.yaml]: wdyt to use the [k8s metadata](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#use-container-fields-as-values-for-environment-variables) information to fill this value?

ex:
```
        - name: RELEASE
          valueFrom:
            resourceFieldRef:
              containerName: {{ .Chart.Name }}
              resource: image
```

Comment (craigmulligan) [reply incubator/sentry-kubernetes/templates/deployment.yaml]: Ah, good point!

**Review by gianrubio — CHANGES_REQUESTED** [review]
please align the table

**Comment by craigmulligan** [comment]
Hey @gianrubio let me know if you'd like me to squash the commits or do anything further. 

Thanks for the review!

**Comment by robdefeo** [comment]
I tried to run this change and got an error when deploying with helm ```Error: release sentry-kubernetes failed: Deployment.apps "sentry-kubernetes" is invalid: spec.template.spec.containers[0].env[2].valueFrom.resourceFieldRef.resource: Unsupported value: "image": supported values: "limits.cpu", "limits.ephemeral-storage", "limits.memory", "requests.cpu", "requests.ephemeral-storage", "requests.memory"``` What version on kubernetes does this need to run on? I am running `"v1.9.6"` but I cant find anything about this in the docs

**Comment by craigmulligan** [comment]
Hey @robdefeo, yea the release value was added to the or with out proper testing, there is a PR to revert the change (#6002).

**Comment by robdefeo** [comment]
@hobochild Perfect, thanks for the information, I'll watch that PR

**Pull request merged**